biome.json

```diff
@@ -151,5 +150,8 @@
         "useValidTypeof": "error"
       }
     }
+  },
+  "organizeImports": {
+    "enabled": false
   }
```

Comment:
Biome organize imports by default and it conflicts with prettier so turned it off. https://biomejs.dev/analyzer/#imports-sorting

should we use Biome to sort import?